sk0-back/src/users/services/users.service.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  ConflictException,
+  Inject,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { User } from '../user.entity';
+import { USER_REPOSITORY } from '../../constants';
+import { CreateUserDto } from '../dtos/create-user.dto';
+
+@Injectable()
+export class UsersService {
+  constructor(@Inject(USER_REPOSITORY) private repo: Repository<User>) {}
+
+  create(user: CreateUserDto): Promise<User> {
+    const createdUser = this.repo.create({ ...user });
+    return this.repo.save(createdUser);
+  }
```

Comment:
I would explicitly say that the functions (or methods) are async in this class, (for `create` and `findAll`) I know that you did not use the await keyword in these methods, but would say that these are async operations and not just inferred.

Reply:
made som refactoring